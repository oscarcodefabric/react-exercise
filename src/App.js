import './App.css'

import { Home } from './pages'

const App = () => (
  <div className="App">
    <Home />
  </div>
)

export default App
