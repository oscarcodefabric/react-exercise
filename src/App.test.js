import React from 'react'
import { shallow } from 'enzyme'
import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'

import App from './App'

const mockStore = configureStore()

const store = mockStore()

describe('App component tests', () => {
  it('renders without crashing', () => {
    const app = shallow(
      <Provider store={store}>
        <App />
      </Provider>
    )

    expect(app).toMatchSnapshot()
  })
})
