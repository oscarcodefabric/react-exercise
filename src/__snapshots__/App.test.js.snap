// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests renders without crashing 1`] = `ShallowWrapper {}`;
